cmd/root.go

```diff
@@ -7,30 +7,16 @@ import (
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "hugo",
-	Short: "Hugo is a very fast static site generator",
-	Long: `A Fast and Flexible Static Site Generator built with
-                love by spf13 and friends in Go.
-                Complete documentation is available at http://hugo.spf13.com`,
+	Use:   "hwsc-dev-ops",
```

Comment:
I will be renaming the repo from `hwsc-dev-ops` to `hwsc-cli` so change the following property.

`Use:   "hwsc-cli",`